ygen/codegen.go

```diff
@@ -367,6 +393,101 @@ func (cg *YANGCodeGenerator) GenerateGoCode(yangFiles, includePaths []string) (*
 	}, nil
 }
 
+// GenerateProto3 generates Protobuf 3 code for the input set of YANG files.
+// The YANG schemas for which protobufs are to be created is supplied as the
+// yangFiles argument, with included modules being searched for in includePaths.
+// Returns a GeneratedProto3 struct containing the messages that are to be
+// output, along with any associated values (e.g., enumerations).
+func (cg *YANGCodeGenerator) GenerateProto3(yangFiles, includePaths []string) (*GeneratedProto3, *YANGCodeGeneratorError) {
+	// TODO(robjs): Handle enumerated types in proto messages.
```

Comment:
Agreed -- I guess we just need to decide how best we should log errors. Perhaps just create an issue here? The plan is to resolve this in a subsequent CL here.

Reply:
I'd probably create an issue and put TODO(some ptr to issue) in general for anything major. I think it's worth having in this case so that someone who comes along doesn't expect the code to be fully functional.

Reply:
ACK -- I created #20 here, which is addressed in a subsequent PR in this stack of them.